asv_benchmarks/asv.conf.json

```diff
@@ -4,30 +4,24 @@
     "version": 1,
 
     // The name of the project being benchmarked
-    "project": "scikit-learn",
+    "project": "asv",
```

Comment:
```suggestion
    "project": "scikit-learn",
```

Reply:
I mixed up sync'ing and ended up overwriting my initial commit. Sorry about it.